draft-ietf-tls-rfc8446bis.md

```diff
@@ -5213,7 +5213,10 @@ provides several recommendations to assist implementors.
 
 TLS requires a cryptographically secure pseudorandom number generator (CSPRNG).
 In most cases, the operating system provides an appropriate facility such
-as /dev/urandom, which should be used absent other (e.g., performance) concerns.
+as /dev/urandom, which should be used absent other (e.g., performance) concerns. On 
+newer Operating Systems, the getrandom interface SHOULD be used as
+it can be used in sandboxing environments.
```

Comment:
I can't suggest this, but here's what I'd say:

> A performant and appropriately-secure CSPRNG is provided
> by most operating systems or can be sourced from a cryptographic library.

Reply:
got it.